pkg/balancer-js/src/utils/errors.ts

```diff
@@ -152,6 +152,7 @@ const balancerErrorCodes: Record<string, string> = {
   '601': 'FLASH_LOAN_FEE_PERCENTAGE_TOO_HIGH',
   '602': 'INSUFFICIENT_FLASH_LOAN_FEE_AMOUNT',
   '603': 'AUM_FEE_PERCENTAGE_TOO_HIGH',
+  '999': 'IMPOSSIBLE',
```

Comment:
```suggestion
  '999': 'SHOULD_NOT_HAPPEN',
```
I've seen this used for exceptions like this; might be clearer/more expected.